XML code review

psteckler opened the pull request:
This is an attempt to merge server-protocol from psteckler's github to main PG repo.

erikmd commented on FAQ.md:
```diff
@@ -68,31 +67,52 @@ Q. I have just installed Emacs, ProofGeneral and a proof assistant.
 
 A. You need to enable Unicode Tokens by the menu item:
 
+<<<<<<< HEAD
```
Hi @psteckler, your copy of the FAQ.md file currently contains 7 "<<<<<<<-delimited" regions. Could you look at this?

psteckler commented:
Thanks for spotting this!

I've removed the merge conflicts, also removed the question about limited Undo, which no longer applies.

There are some other questions that need updating, such as the mention of the now-defunct Trac, which I haven't fixed.

erikmd commented:
Hi @psteckler, Thanks for your commit!

> There are some other questions that need updating, such as the mention of the now-defunct Trac, which I haven't fixed.

OK. Anyway in the long run, as was suggested by @Matafou IIRC, we could create a GitHub wiki for PG with various tips, and most of the FAQ.md questions could be migrated to that place as well.

erikmd commented on Makefile.devel:
```diff
@@ -84,16 +84,17 @@ DEVELOPERS=\
-PRERELEASE_TAG=4.4.1~pre
-### Formerly: PRERELEASE_TAG=4.4pre$(shell date "+%y%m%d")
+PRERELEASE_PREFIX=5\.0~pre
+PRERELEASE_TAG=5.0~pre
```
@psteckler
as discussed in https://github.com/ProofGeneral/PG/pull/127#discussion_r94273448, from the upcoming release it would be better to use a "-git" version suffix for the development version of PG. So could you change this line to `5.0-git` and likewise for other occurrences of PG's version? (in README.md and proof-site.el)

psteckler replied:
Done.

erikmd replied:
Thank you @psteckler!
I had only mentioned README.md and proof-site.el in my comment, but a few other files should also be updated to 5.0-git : to sum up, could you update these as well?
https://github.com/psteckler/ProofGeneral/blob/server-protocol/Makefile.devel#L95-L96
https://github.com/psteckler/ProofGeneral/blob/server-protocol/etc/ProofGeneral.spec#L3
https://github.com/psteckler/ProofGeneral/blob/server-protocol/doc/ProofGeneral.texi#L60
https://github.com/psteckler/ProofGeneral/blob/server-protocol/doc/PG-adapting.texi#L54

psteckler replied:
I've changed these other files now.

erikmd commented on proof-general-pkg.el:
```diff
@@ -0,0 +1,2 @@
+(define-package "proof-general" "4.4.1~pre" "A generic front-end for proof assistants (interactive theorem provers)"
```
@psteckler 
I think the version string in this file (that comes from Clément's PR #157) can also be changed to 5.0-git in your fork

psteckler replied:
@erikmd I've made the version change, thanks.

Matafou commented:
Should we merge now?

Matafou commented:
Or do we still wait about licensing?

psteckler commented:
I'm supposed to meet with MIT TLO tomorrow. Let's wait for the result of
that.

On Tue, Jun 6, 2017 at 8:04 AM, Pierre Courtieu <notifications@github.com>
wrote:

> Or do we still wait about licensing?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ProofGeneral/PG/pull/153#issuecomment-306465788>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/ACJAKEKGVX3sJ7Z5AE8YHYRPwWClbZUMks5sBUBigaJpZM4MCXwv>
> .
>

erikmd commented:
Closing this PR without merging for now (as PG/xml will be integrated as a separate patch).
